apps/src/templates/sectionProgressV2/LegendItem.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './section-progress-refresh.scss';
+import {BodyThreeText} from '@cdo/apps/componentLibrary/typography';
+import color from '@cdo/apps/util/color';
+import FontAwesome from '../FontAwesome';
+import ProgressBox from '../sectionProgress/ProgressBox';
+import {NOT_STARTED, VIEWED, NEEDS_FEEDBACK, FEEDBACK_GIVEN} from './IconKey';
+
+export default function LegendItem({
```

Comment:
Have you thought about how we can reuse this functionality for the data boxes? We will want to show the exact same icons for the exact same types. It would be nice to be able to reuse the code. Maybe we need to have a LegendItem that has the item/icon AND text and a second object (ProgressDataIcon?) that picks the icon/creates the image? Open to whatever though.

Reply:
That's a good question. Not addressed here at the moment, we can refactor as needed when the time comes.